### DISCUSSION TITLE
v1.0.2.7

### DISCUSSION THREAD

**Discussion by PeterSuh-Q3** [PeterSuh-Q3/tinycore-redpill#37] [Q&A] [answered]
1.0.2.5 Provides menu option to disable i915 module loading to prevent console blackout in ApolloLake (DS918+), GeminiLake (DS920+), and Epyc7002 (SA6400)
    1.0.2.6 Added multilingual support languages (locales) (Arabic, Hindi, Hungarian, Indonesian, Turkish)
    1.0.2.7 dbgutils Addon Add/Delete selection menu

<hr /><em>This discussion was created from the release <a href='https://github.com/PeterSuh-Q3/tinycore-redpill/releases/tag/v1.0.2.7'>v1.0.2.7</a>.</em>

**Comment by gericb** [comment, marked as answer]
Oh GREAT!  I noticed this "Blackout" on my Precision 3630 Tower, just assumed it was the norm with any loader (mine 920+) in general, until I did a test build on a XPS 8950, and the screen stayed on all the time.  A minor issue in the grand scheme of things, but nice to have it resolved.  Thank YOU!
